dev/com.ibm.ws.jaxrs.2.1.cdi.2.0_fat/test-applications/atinjectapp/src/jaxrs21/fat/atinject/AtInjectDirectService2.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package jaxrs21.fat.atinject;
+
+import static jaxrs21.fat.atinject.AtInjectApp.toResponse;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/direct2")
+@Dependent
```

Comment:
Could you add a comment here (or as javadoc to the class, etc.) that indicates that public no-arg constructors are not required when the scope is `@Dependent`, but that it is for any other "normal" scope as defined by the spec.